### PR TITLE
Animal table

### PR DESCRIPTION
**Pull request by kamranasif506**
In this PR

- Animal Table created in vet_clinic database
- Data inserted into animal table
- Following Quries run to fetch records
  - Find all animals whose name ends in "mon".
Screenshot 2023-08-15 005430

  - List the name of all animals born between 2016 and 2019.
Screenshot 2023-08-15 005757

  - List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot 2023-08-15 010003

  - List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot 2023-08-15 011721

  - List name and escape attempts of animals that weigh more than 10.5kg
Screenshot 2023-08-15 010603

  - Find all animals that are neutered.
Screenshot 2023-08-15 010701

  - Find all animals not named Gabumon.
Screenshot 2023-08-15 010813

  - Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot 2023-08-15 011114

### CODE REVIEW

**Review by V-Blaze — CHANGES_REQUESTED** [review]
.

File: README.md [hunk 1]
```diff
@@ -102,123 +42,33 @@ After you're finished please remove all the comments and instructions!
...
+👤 **Kamran Asif**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/kamranasif506)
+- Twitter: [@twitterhandle](https://twitter.com/kamiasif55)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kamran-asif-b3a612130/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (V-Blaze) [inline README.md]: - You have done great so far customizing your readme, but there are few changes you still need to make. :+1: :+1: 
- [ ] Looking at the screenshot below, you can see you still have some placeholder comments on the readme, you are required to remove them all as they are just there to guide you. :+1: :+1: 

- - Screenshot:
Selection_002

File: README.md [hunk 2]
```diff
@@ -258,34 +110,19 @@ If you like this project...
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (V-Blaze) [inline README.md:121]: - [ ] The license link leads to a 404 error page, I'll suggest you add a LICENSE file and link it correctly. :+1: :+1: 

- - Screenshot:
Screenshot from 2023-08-14 21-42-39

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @kamranasif506 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


## Highlights
- Correct data inserted into the database✔✔
- Queries are written correctly :heavy_check_mark: 
- Screenshots show correct results :heavy_check_mark: 

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (KanzaTahreem) [inline README.md:1]: - [OPTIONAL] Good job for implementing the requested changes 🥇  I've noticed that the **_Getting Started_** section is currently missing from your README. I highly recommend adding it to provide users with a clear starting point. For reference, you can take a look at  [this README](https://github.com/KanzaTahreem/Vet-Clinic-Database) ⛏✨

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @kamranasif506 

Good job for implementing the suggested changes💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Table is created
-  ✅ Screenshots are added 
-  ✅ Successful queries

## Optional suggestions ❕

_Please go through the comments under review_

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**